insights/parsers/netstat.py

```diff
@@ -182,28 +177,31 @@ def parse_content(self, content):
                             else:
                                 first_layer[key] = val.strip().lower()
                     else:
-                        # To deal with lines look like:
-                        # 0 bad segments received.
+                        # To deal with lines that look like:
+                        # '0 bad segments received.'
+                        # or
+                        # 'destination unreachable: 107'
+                        # or
+                        # 'Quick ack mode was activated 9 times'
                         if has_scd_layer:
                             first_layer[layer_key] = second_layer
                             has_scd_layer = False
                             second_layer = {}
-                        data = line.split()
-
-                        # Some line's end has a '.', it'll be removed
-                        tmp_data = data[-1]
-                        if tmp_data[-1] == ".":
-                            data[-1] = tmp_data[:-1]
-                        for d in data:
-                            if d.isdigit():
-                                val = d
+
+                        # Some lines end with '.', trim that off
+                        if line.endswith('.'):
+                            line = line[:-1]
+
+                        parts = line.split()
+
+                        found_val = None
+                        for val in parts:
+                            if val.isdigit():
+                                found_val = val
                                 break
-                        else:
-                            # Line contained no number, ignore
-                            continue
-                        data.remove(val)
-                        key = "_".join([k.lower() for k in data])
-                        first_layer[key] = val
+                        if found_val is not None:
+                            key = '_'.join(k.lower() for k in parts if k != found_val)
+                            first_layer[key] = found_val
```

Comment:
Would you raise a "ParseException" here for `found_val` is None?

Reply:
According to http://insights-core.readthedocs.io/en/latest/exception_model.html#parsing-and-exceptions we should only raise a ParseException if the entire output is unusable.  This change specifically removes the previous ParseException, which was counter to that policy.

Reply:
Okay, makes sense to me.